meta/runtime.yml

```diff
@@ -3,7 +3,7 @@ requires_ansible: '>=2.9.10'
 
 action_groups:
   vmware:
-  - vmware_dns_config
+  - vcenter_domain_user_group_info
```

Comment:
I understand you moved `vmware_dns_config` to preserver the alphanumeric order. Can you also move `vcenter_domain_user_group_info` at the right place?